plenum/server/node.py

```diff
@@ -406,6 +395,97 @@ def __init__(self,
         # We need future_primaries to calculate applied primaries correctly
         self.future_primaries_handler = FuturePrimariesBatchHandler(self.db_manager, self)
 
+    def init_req_managers(self):
+        self.write_manager = WriteRequestManager(self.db_manager)
+        self.read_manager = ReadRequestManager()
+        self.action_manager = ActionRequestManager()
+
+    def init_storages(self, storage=None):
+
+        # Config ledger and state init
+        self.db_manager.register_new_database(CONFIG_LEDGER_ID,
+                                              self.init_config_ledger(),
+                                              self.init_config_state())
+
+        # Pool ledger init
+        self.db_manager.register_new_database(POOL_LEDGER_ID,
+                                              self.init_pool_ledger(),
+                                              self.init_pool_state())
+
+        # Domain ledger init
+        self.db_manager.register_new_database(DOMAIN_LEDGER_ID,
+                                              storage or self.init_domain_ledger(),
+                                              self.init_domain_state())
+
+        # Audit ledger init
+        self.db_manager.register_new_database(AUDIT_LEDGER_ID,
+                                              self.init_audit_ledger())
+
+    def init_bls_bft(self):
+        self.bls_bft = self._create_bls_bft()
+
+    def init_common_managers(self):
+        # Pool manager init
+        HasPoolManager.__init__(self, self.poolLedger,
+                                self.states[POOL_LEDGER_ID],
+                                self.write_manager,
+                                self.ha,
+                                self.cliname,
+                                self.cliha)
+
+    def register_req_handlers(self):
+        self.register_pool_req_handlers()
+        self.register_domain_req_handlers()
+        self.register_config_req_handlers()
+        self.register_audit_req_handlers()
+        self.register_action_req_handlers()
+
+    def register_audit_req_handlers(self):
+        pass
+
+    def register_domain_req_handlers(self):
+        nym_handler = NymHandler(self.config, self.db_manager)
+        get_txn_handler = GetTxnHandler(self, self.db_manager)
```

Comment:
GET_TXN is not Domain-specific request. Maybe move it to `register_common_req_handlers`?

Reply:
i agree.